run_pylint.sh

```diff
@@ -140,72 +129,42 @@ check_subtest() {
            --output-format="colorized" \
            --rcfile=/dev/null \
            --msg-template="$MSGFMT" "${WHAT}"
-    RET="$?"
-    if [ "$RET" -ne "0" ] && [ "$FF" -eq "1" ]
-    then
-        record_return 1
-        return $RET
-    elif [ "$RET" -ne "0" ]
-    then
-        record_return 1
-    else
-        # Just print FIXME/TODO warnings, don't fail on them.
-        pylint -rn --init-hook="$SUBTESTINIT_HOOK" \
+    # Just print FIXME/TODO warnings, don't fail on them.
+    pylint -rn --init-hook="$SUBTESTINIT_HOOK" \
                --disable=all \
                --enable=W0511 \
                --output-format="colorized" \
                --rcfile=/dev/null \
                --msg-template="$MSGFMT" "${WHAT}"
-    fi
     if [ -n "$PEP8" ]
     then
-        $PEP8 --ignore=$PEP8IGNORE "$WHAT"
-        RET="$?"
-        if [ "$RET" -ne "0" ] && [ "$FF" -eq "1" ]
-        then
-            record_return 1
-            return $RET
-        elif [ "$RET" -ne "0" ]
-        then
-            record_return 1
-        fi
+        run_ff $PEP8 --ignore=$PEP8IGNORE "$WHAT"
     fi
 }
 
 check_subtests() {
     for thing in pretests subtests intratests posttests
     do
-        trap "break" INT
```

Comment:
I'd like to keep this here because it allows ctrl-c'ing out of the loop to abort the run.  Without this, it's most likely to kill pylint instead of the for loop (b/c more time spent in pylint), which means the remaining pylint runs just keep going.   Only other way out is to spam ctrl-c and get lucky with the timing.